src/main/java/io/github/zerthick/protectionperms/ProtectionPerms.java

```diff
@@ -78,13 +78,18 @@ public void onServerInit(GameInitializationEvent event) {
                 logger.warn("Error loading default config! Error: " + e.getMessage());
             }
         }
-
         //Load invert
         try {
-            CommentedConfigurationNode linksNode = configLoader.load().getNode("invert");
-            boolean invert = linksNode.getBoolean(false);
-            PermHandler.getInstance().init(invert);
-
+            CommentedConfigurationNode modeNode = configLoader.load().getNode("mode");
+            if (modeNode.isVirtual()) {
+                logger.warn("Config corrupt! Using defaults (whitelist mode)\nDelete " + instance.getId() + ".conf to fix");
+            }
+            String mode = modeNode.getString("whitelist");
+            if (mode.equals("whitelist")) {
+                PermHandler.getInstance().init(false);
+            } else if (mode.equals("blacklist")){
+                PermHandler.getInstance().init(true);
+            }
```

Comment:
There should be an else block here in case the user types something other than "whitelist" and "blacklist", we should also probably display some kind of warning as well

Reply:
I agree. Should the mode be set to `whitelist` if the mode does not equal one of them?

Reply:
I added an else block.
Please see dec5390 for important notes.

Reply:
Yeah whitelist should be the default